src/codec/codec.h

```diff
@@ -37,13 +37,11 @@ static constexpr uint8_t SIZE_LENGTH = 4;
 static constexpr uint8_t HEADER_LENGTH = VERSION_LENGTH + SIZE_LENGTH;
 static constexpr uint32_t UINT24_MAX = (1 << 24) - 1;
 
-struct RowContext;
 class RowBuilder;
 class RowView;
 class RowProject;
 
 // TODO(wangtaize) share the row codec context
```

Comment:
delete `TODO` comment as well

Reply:
Just deleted it, so sorry, I wasn't sure if deleting it or not